app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,13 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-
+    <% facade = HeaderFacade.new %>
```

Comment:
confused why we call this a facade. Is this like a homepage? Facade is the envelope of a building. Outward facing. Is that what they are going for? First impression of the user?